pkgs/development/python-modules/python3-msrplib/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, isPy3k
+, buildPythonPackage
+, fetchFromGitHub
+, python3-application
+, python3-eventlib
+, python3-gnutls
+, twisted
+}:
+
+buildPythonPackage rec {
+  pname = "python3-msrplib";
+  version = "0.20.1";
+
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "AGProjects";
+    repo = pname;
+    rev = "5bd069620d436d5a65e1c369e43cc6b88857fb9e";
```

Source is [tagged](https://github.com/AGProjects/python3-msrplib/releases/tag/0.21.0).

```suggestion
    rev = "v${version}";
```